AUTHORS.csv

```diff
@@ -0,0 +1,249 @@
+"Full Name";                            "Surname First"; 						"Name Reference"
```

Comment:
If this file was really comma-separated rather than semicolon-plus-space, I think github would display it as a sortable, searchable table.

Reply:
Good shout - semicolons were for speed as CSV didn't like commas within the text, but I'll try escaping them. Likewise spacing was to make it easier to edit the pure text file, but if github will render intelligently it would be worth removing these I think.

Reply:
Seems OK to avoid extra spaces, people can use all kinds of plugins in text editors or IDEs to edit a CSV file, or even a graphical spreadsheet program as long as they are careful when saving.

Reply:
Changed, and github does indeed render it nicely!